src/main/groovy/com/lesfurets/jenkins/unit/declarative/AllOfDeclaration.groovy

```diff
@@ -0,0 +1,49 @@
+package com.lesfurets.jenkins.unit.declarative
+
+import org.springframework.util.AntPathMatcher
+
+class AllOfDeclaration extends WhenDeclaration {
+
+    List<String> branches = []
+    List<Boolean> expressions = []
+
+    def branch(String name) {
+        this.branches.add(name)
+    }
+
+    def expression(Closure closure) {
+        this.expressions.add(closure)
+    }
+
+    def expressions(delegate) {
+        def exp_result;
+        for (def exp in this.expressions) {
+            exp_result = executeWith(delegate, exp)
+            if (!exp_result) {
+                return false
+            }
+        }
+        return true
+    }
+
+    Boolean execute(Object delegate) {
```

Comment:
Please use the primitive `boolean` type here.

Reply:
I can't change this without changing the return type of `WhenDeclaration.execute`